@ecomplus/storefront-template/template/js/lib/utils.js

```diff
@@ -91,3 +91,39 @@ if ($('#popup-modal').length) {
     }
   })
 }
+
+if ($('.multiple-collection-shelf').length) {
+  const $multipleCollections = $('.multiple-collection-shelf')
+  for (const key in $multipleCollections) {
+    if (Object.hasOwnProperty.call($multipleCollections, key)) {
+      const { dataset } = $multipleCollections[key]
+      if (dataset && dataset.id) {
+        const $multipleCollection = $(`.multiple-collection-shelf[data-id="${dataset.id}"]`)
+        const $multipleTabList = $multipleCollection.find('.products-carousel-tabs__list')
+        $multipleTabList.click((e) => {
+          if (e.target && e.target.dataset && e.target.dataset.tabTitle) {
+            const titleTab = e.target.dataset.tabTitle
+            const $tabs = $multipleTabList[0].children
+            for (const tabCollectionTitle in $tabs) {
+              const tab = $tabs[tabCollectionTitle]
+              if (tab.children) {
```

Comment:
O que é possível aqui é que `tab` seja undefined, nesse caso daria fatal de qualquer forma por `chindren of undefined`, se `tab` não for undefined com certeza ela é um `HTMLElement`, e aí com certeza ela tem o array like `.children`, pode ser vazio mas a propriedade existe.
Esse `if (tab.children)` é inútil...